src/mixins/meta.ts

```diff
@@ -127,3 +155,51 @@ export function metaTag(
 
   return metaTagsObject;
 }
+
+// ALernate locales generator
+export interface AlterateLocalLink {
+  href: string;
+  hreflang: string;
+  rel: "alternate";
+}
+
+export interface OpenGraphMetaTag {
+  content: string;
+  property: string;
+}
+
+function pageAlternateLocales(locales: QMetaI18nAlternateLocale[], useOpengraph?: QMetaI18nAlternateLocale) {
```

Comment:
`useOpengraph` sottintende che sia un parametro booleano.
Cambierei in `openGraphOptions` o roba simile.

Come mai serve un locale diverso per OpenGraph? In generale, perchè la roba di OpenGraph è dietro un flag e non è direttamente aggiunta di default?

Reply:
As we said yesterday by phone, I initially wanted to let people choose if they wanted to support Open Graph or not by adding a boolean flag. However, we came to conclusion that this package should enforce some best practice to follow and thus I changed the name of the flag.

The additional locale is required by OpenGraph since this is the standard method through which it detects the locale of the page. Usually Google and search engines base themselves on the lang attribute of the <html> tag, but OpenGraph wants a dedicated meta tag inside the page.